apt-cacher/Dockerfile

```diff
@@ -0,0 +1,16 @@
+#
+# Build: docker build -t apt-cacher .
+# Run: docker run -d -p 3142:3142 --name apt-cacher-run apt-cacher
+#
+# and then you can run containers with:
+#   docker run -t -i --rm -e http_proxy http://dockerhost:3142/ debian bash
+#
+# Here, `dockerhost` is the IP address or FQDN of a host running the Docker daemon
+# which acts as an APT proxy server.
+FROM        ubuntu
+
+VOLUME      ["/var/cache/apt-cacher-ng"]
```

Comment:
@pietrushnic I can see we already have a `docker-compose` for this one. I think it would make sense to:
- move volume definition to `docker-compose`
- get rid of `EXPOSE` from Dockerfile - the port it is already exposed in the `docker-compose`
- environment variable from the comments above (`http_proxy`) to `envirornoment` section in `docker-compose` file
- update the README to use the `docker-compose` do build and run the container